redis.conf

```diff
@@ -1574,6 +1574,11 @@ lua-time-limit 5000
 #
 # cluster-allow-reads-when-down no
 
+# This option, when set to yes, allows nodes to serve pubsub local traffic while the
+# the cluster is in a down state, as long as it believes it owns the slots.
```

Comment:
I think we should at least describe the implications of setting this as yes or no.

Reply:
Added some info.